### PR TITLE
fix: toggleButtonRestyling

### PR DESCRIPTION
**Pull request by palbizu**
## Description
General Toggle Button was re-styled (reduction of size), in this way the default styles of material were changed.

### Testing
Visual Testing

Evidence;

image

### Checklist:
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### Documentation
Make sure that you have documented corresponding changes in this repository or [hypertrace docs repo](https://github.com/hypertrace/hypertrace-docs-website) if required.

### CODE REVIEW

**Comment by anandtiwary** [comment]
please share before and after screenshot

**Comment by palbizu** [comment]
@anandtiwary, added

File: projects/components/src/toggle-switch/toggle-switch.component.scss [hunk 1]
```diff
@@ -3,8 +3,14 @@
...
 :host {
   ::ng-deep {
+    .mat-slide-toggle {
+      height: 20px;
+      line-height: 20px;
+    }
+
     .mat-checked {
       .mat-slide-toggle-thumb-container {
+        transform: translate3d(10px, 0, 0) !important;
```
Comment (aaron-steinfeld) [inline projects/components/src/toggle-switch/toggle-switch.component.scss:13]: is the `!important` needed here? We generally want to save that for a tool of last resort

Comment (palbizu) [reply projects/components/src/toggle-switch/toggle-switch.component.scss:13]: I checked the material documentation and did not find a better solution, I know that it's not the best solution.

Comment (arjunlalb) [reply projects/components/src/toggle-switch/toggle-switch.component.scss:13]: @palbizu do we want this smaller dimension toggles everywhere? Could you please check with Michael about this? If not, we'll have to introduce a size param in the component.

Comment (palbizu) [reply projects/components/src/toggle-switch/toggle-switch.component.scss]: @arjunlalb It was validated with Michael the changes effectively apply to all toggle buttons

**Review by aaron-steinfeld — APPROVED** [review]

**Pull request merged**